netlify.toml

```diff
@@ -4,18 +4,30 @@
 
 # Temporary header config until Netlify supports setting `headers` on `next.config.js`
 # https://github.com/netlify/netlify-plugin-nextjs/issues/150
+
 [[headers]]
   for = "/*"
   [headers.values]
     X-Frame-Options = "DENY"
     X-XSS-Protection = "1; mode=block"
     X-Content-Type-Options = "nosniff"
     Referrer-Policy = "same-origin"
-    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com media-library.cloudinary.com analytics.google.com 'unsafe-eval'; frame-src 'self' cloudinary.com; img-src 'self' blob: data: https:; frame-ancestors 'none';"
+    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com analytics.google.com media-library.cloudinary.com; frame-src 'self' cloudinary.com; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
     Permissions-Policy = "camera=(), microphone=(), geolocation=(), interest-cohort=()"
+
 [[headers]]
-  for = "/_next/static/*"
+  for = "/admin/*"
   [headers.values]
+    X-Frame-Options = "DENY"
+    X-XSS-Protection = "1; mode=block"
+    X-Content-Type-Options = "nosniff"
+    Referrer-Policy = "same-origin"
+    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com 'unsafe-eval'; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
```

Comment:
`media-library.cloudinary.com;` adn `cloudinary.com;` is required on CMS, it's used by upload image

Reply:
okayy,,
please check it agian